letsencrypt-nginx/letsencrypt_nginx/configurator.py

```diff
@@ -213,17 +216,13 @@ def choose_vhost(self, target_name):
         :rtype: :class:`~letsencrypt_nginx.obj.VirtualHost`
 
         """
-        vhost = None
-
         matches = self._get_ranked_matches(target_name)
         if not matches:
-            # No matches. Create a new vhost with this name in nginx.conf.
-            filep = self.parser.loc["root"]
-            new_block = [['server'], [['server_name', target_name]]]
-            self.parser.add_http_directives(filep, new_block)
-            vhost = obj.VirtualHost(filep, set([]), False, True,
-                                    set([target_name]), list(new_block[1]))
-        elif matches[0]['rank'] in xrange(2, 6):
+            logger.info("Virtual Host not found for %s", target_name)
```

Comment:
We could expand this to say `"Virtual Host not found for %s. Are you sure you're really serving that domain name?"`  It could in the future make sense to create transient vhosts in this case, for instance to support a case like `letsencrypt --authenticator nginx --installer exim -d mail.example.com`, even if nginx only answers for `www.example.com`.  But that may be a low urgency case?